bin/set-up-current-account.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -euo pipefail
+
+# Name the account based on the current account alias
+ACCOUNT_ALIAS="$(./bin/current-account-alias.sh)"
+ACCOUNT_ID="$(./bin/current-account-id.sh)"
+REGION="$(./bin/current-region.sh)"
+BACKEND_CONFIG_FILE="$ACCOUNT_ALIAS.s3.tfbackend"
+
+# Get project name
+terraform -chdir=infra/project-config refresh > /dev/null
+PROJECT_NAME=$(terraform -chdir=infra/project-config output -raw project_name)
+
+TF_STATE_BUCKET_NAME="$PROJECT_NAME-$ACCOUNT_ID-$REGION-tf"
+TF_STATE_KEY="infra/account.tfstate"
+
+echo "=================="
+echo "Setting up account"
+echo "=================="
+echo "ACCOUNT_ALIAS=$ACCOUNT_ALIAS"
+echo "ACCOUNT_ID=$ACCOUNT_ID"
+echo "PROJECT_NAME=$PROJECT_NAME"
+echo "TF_STATE_BUCKET_NAME=$TF_STATE_BUCKET_NAME"
+echo "TF_STATE_KEY=$TF_STATE_KEY"
+echo "REGION=$REGION"
+echo
+echo "------------------------------------------------------------------------------"
+echo "Bootstrapping the account by creating an S3 backend with minimal configuration"
+echo "------------------------------------------------------------------------------"
+echo 
+echo "Creating bucket: $TF_STATE_BUCKET_NAME"
+aws s3api create-bucket --bucket $TF_STATE_BUCKET_NAME --region $REGION > /dev/null
+echo
+echo "----------------------------------"
+echo "Creating rest of account resources"
+echo "----------------------------------"
+echo 
+
+cd infra/accounts
+
+# Create the infrastructure for the terraform backend such as the S3 bucket
+# for storing tfstate files and the DynamoDB table for tfstate locks.
+# -reconfigure is used in case this isn't the first account being set up
+# and there is already a .terraform directory
+terraform init \
+  -reconfigure \
+  -input=false \
+  -backend-config="bucket=$TF_STATE_BUCKET_NAME" \
+  -backend-config="key=$TF_STATE_KEY" \
+  -backend-config="region=$REGION"
+
+# Import the S3 bucket that was created in the previous step so we don't recreate it
+terraform import module.backend.aws_s3_bucket.tf_state $TF_STATE_BUCKET_NAME
```

Comment:
If the bucket already exists when this script is run you will take an error on import.

Reply:
What do you suggest? Should we do a check to see if the bucket exists at the beginning of the script?

Reply:
It technically didn't break anything so something as simple as "this is expected if you already created this resource" above the error would be sufficient. If not we would want to check that both the bucket exists and the backend file exists and matches the bucket  before trying to direct them to the best course of action.

Reply:
Hmm, I'm not following. How do I reproduce the issue you're describing? Are you referring to the situation where someone runs `set-up-current-account.sh` twice, or the situation where by some coincidence there's already a bucket with the same name that some other person in the world created, and now the name conflicts?

Reply:
You can reproduce the issue by running the `set-up-current-account.sh` script more than once. The most likely scenario probably be the same person running it twice.

Reply:
I see, I get this:

<img width="750" alt="image" src="https://github.com/navapbc/template-infra/assets/447859/78060a95-2df0-451b-83e5-efd1d8d29264">

seems like a reasonable error for now, but I'll create a follow-up ticket to be a little more defensive in case someone runs into this.

Reply:
https://github.com/navapbc/template-infra/issues/268